buildconfig/Jenkins/Conda/nightly_build_and_deploy.jenkinsfile

```diff
@@ -53,6 +53,18 @@ pipeline {
     WIN_BASH = "C:\\Program Files\\git\\bin\\bash.exe"
   }
   stages {
+    // Store the git commit hash in an environment variable so that we can use
+    // the same commit for all stages of the pipeline. This is perfomed by using
+    // the following step at the start of each subsequent stage:
+    //   checkout scm: [$class: 'GitSCM', branches: [[name: "${GIT_SHA}"]]]
```

Comment:
Do we need to keep this commented out `checkout` command?

Reply:
I put it there to help explain why we are using that command in each of the other sections.

Reply:
Oh yes I see now I have read the whole thing properly. Thanks.